CONTRIBUTING.md

```diff
@@ -2,17 +2,19 @@
 
 In order to build `bootc` you will need the following dependencies.
 
-Fedora: 
-```
-sudo dnf install ostree-libs ostree-devel
+Fedora:
+
+```bash
+sudo dnf install clippy openssl-devel ostree-devel ostree-libs rustfmt
```

Comment:
As of recently for Fedora one can now also just `dnf builddep bootc`, but this is fine as is.